<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Sebastian Pipping</author>
 <class>SearchReplaceWidget</class>
 <widget class="QWidget" name="SearchReplaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>144</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbReplacePattern">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>9</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Text to replace with</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lFind">
       <property name="text">
        <string>Fin&amp;d:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>cbFindPattern</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lMode">
       <property name="text">
        <string>&amp;Mode:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>cbMode</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbFindPattern">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>9</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Text to search for</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lReplace">
       <property name="text">
        <string>Rep&amp;lace:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>cbReplacePattern</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,1,0,0,0,0,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QComboBox" name="cbMode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Search mode</string>
         </property>
         <property name="maxVisibleItems">
          <number>4</number>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Plain text</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Whole words</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Escape sequences</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Regular expression</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbMatchCase">
         <property name="toolTip">
          <string>Match case sensitive</string>
         </property>
         <property name="icon">
          <iconset theme="format-text-superscript">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbSelectionOnly">
         <property name="toolTip">
          <string>Search in the selection only</string>
         </property>
         <property name="icon">
          <iconset theme="edit-select-all">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="tbFindNext">
         <property name="toolTip">
          <string>Jump to next match</string>
         </property>
         <property name="icon">
          <iconset theme="go-down-search">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbFindPrev">
         <property name="toolTip">
          <string>Jump to previous match</string>
         </property>
         <property name="icon">
          <iconset theme="go-up-search">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bReplaceNext">
         <property name="toolTip">
          <string>Replace next match</string>
         </property>
         <property name="text">
          <string>&amp;Replace</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="searchCancelStacked">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="searchPage">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="bReplaceAll">
             <property name="toolTip">
              <string>Replace all matches</string>
             </property>
             <property name="text">
              <string>Replace &amp;All</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bFindAll">
             <property name="text">
              <string>&amp;Find All</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="cancelPage">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tbSwitchFindReplace">
         <property name="toolTip">
          <string>Switch to incremental search bar</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="games-config-options">
           <normaloff>.</normaloff>.</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="bCancel">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="dialog-close"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbFindPattern</tabstop>
  <tabstop>cbReplacePattern</tabstop>
  <tabstop>cbMode</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
